content/SCALE/SCALECLIReference/System/CLIKMIP.md

```diff
@@ -12,8 +12,187 @@ tags:
+| PROTOCOL_TLSv1_2 | PROTOCOL_TLSv1_2 |
++------------------+------------------+
+```
+{{< /nest-expand >}}
+{{< /expand >}}
+
+### Sync_Keys command
+Use the `sync_keys` command to sync ZFS/SED keys between KMIP the server and TN SCALE database.
+
+{{< expand "Using the Sync_keys Command" "v" >}}
+#### Description
+The `sync_keys` command does not require entering a property argument.
+Enter the command then press <kbd>Enter</kbd>.
+The command returns an empty line. Use the `system kmip kmip_sync_pending` command to verify there is no longer a sync operation pending.
+
+#### Usage
+From the CLI prompt, enter:
+
+<code>system kmip sync_keys</code>
+
+{{< nest-expand "Command Example" "v" >}}
+```
+system kmip sync_keys
+
+```
+{{< /nest-expand >}}
+{{< /expand >}}
+
+#### Update Command
+Use the `update` command to update the KMIP server settings.
+
+{{< expand "Using the Update Command" "v" >}}
+#### Description
+The `update` command has 11 property options. See **Update Properties** for details.
+Enter the property argument using the `=` delimiter to separate property and value. Double-quote values that include special characters.
+Enter the command string then press <kbd>Enter</kbd>.
+The command returns an empty line. Enter the [`system kmip config`](#) command to verify settings.
+
+{{< nest-expand "Update Properties" "v" >}}
+{{< truetable >}}
+| Property | Description | Syntax Example |
+|----------|----------|-------------|----------------|
+| `enabled` | Set to `true` to activate the KMIP configuration and begin syncing keys with the KMIP server. `enabled` if true, cannot be set to disabled if there are existing keys pending to be synced. However users can still perform this action by enabling `force_clear`. | <code>enabled="<i>true/false</i>"</code> |
+| `manage_sed_disks` | Set to `true` to enabled and manage syncs keys from local database to remote KMIP server. Enabling this option allows the key server to manage creating or updating the global SED password, creating or updating individual SED passwords, and retrieving SED passwords when SEDs are unlocked. When set to `false`, if there are any keys left to be retrieved from the KMIP server, it syncs them back to local database. Disabling this option leaves SED password management with the local system. | <code>manage_sed_disks="<i>true/false</i>"</code> |
+| `manage_zfs_keys` | Set to `true` enabled and syncs keys from local database to remote KMIP server. Use the KMIP server to manage ZFS encrypted dataset keys. The key server stores, applies, and destroys encryption keys whenever an encrypted dataset is created, when an existing key is modified, an encrypted dataset is unlocked, or an encrypted dataset is removed. When set to `false`, if there are any keys left to be retrieved from the KMIP server, it syncs them back to local database. Unsetting this option leaves all encryption key management with the local system. | <code>manage_zfs_keys="<i>true/false</i>"</code> |
```

Comment:
I'm a little confused with what the first sentence of each of these is trying to say. Could use another pass to clarify

Reply:
Updated the descriptions.